project1/css/index.css

```diff
@@ -62,3 +62,102 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+nav {
+	padding-top:2%;
```

Comment:
I would use `px` here instead of percentages for padding. `px` is a fixed-size and will stay the same size in any device height, whereas `%` will change depending on device height and may not be the padding size you are expecting.